web/server/codechecker_server/database/db_cleanup.py

```diff
@@ -190,3 +191,23 @@ def upgrade_severity_levels(session_maker, severity_map):
         except (sqlalchemy.exc.OperationalError,
                 sqlalchemy.exc.ProgrammingError) as ex:
             LOG.error("Failed to upgrade severity levels: %s", str(ex))
+
+
+def upgrade_severity_levels(session_maker, severity_map):
```

Comment:
It is a little bit strange for me that in this file we have two functions with the same name, one is "private" and one is public.

By the way there are two more places where we need to chunk the data:
https://github.com/Ericsson/codechecker/blob/169146e2b72a000030033f47b2dd6fb9438ab80b/web/server/codechecker_server/api/report_server.py#L1305-L1314
https://github.com/Ericsson/codechecker/blob/169146e2b72a000030033f47b2dd6fb9438ab80b/web/server/codechecker_server/api/report_server.py#L2409-L2411

For this reason maybe it would be better to create a generic chunk function similar to this: https://dev.to/orenovadia/solution-chunked-iterator-python-riddle-3ple
```py
def chunks(iterator: Iterator[T], n: int) -> Iterator[Iterator[T]]:
    for first in iterator: # take one item out (exits loop if `iterator` is empty)
        rest_of_chunk = itertools.islice(iterator, 0, n - 1)
        yield itertools.chain([first], rest_of_chunk)  # concatenate the first item back

x = {10:0, 20:0, 30:0, 40:0, 50:0, 60:0, 70:0}
for chunk in chunks(iter(x), 3):
    print(list(chunk))
```
What do you think @bruntib?

Reply:
Good idea, I like it :) Thanks